wrapper/src/main/java/software/amazon/jdbc/profile/ConfigurationProfilePresetCodes.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.profile;
+
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+public class ConfigurationProfilePresetCodes {
+
+  // Presets family A, B, C - no connection pool
+  // Presets family D, E ,F - internal connection pool
+  // Presets family G, H, I - external connection pool
+
+  public static final String A0 = "A0"; // Normal
```

Comment:
i think we should maybe prefix all the profile variables, something like PROFILE_A0 ?